scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+  - name: Change NODENAME in RabbitMQ settings
+    lineinfile:
+      path: /etc/rabbitmq/rabbitmq-env.conf
+      regexp: "NODENAME=.*"
+      line:  "NODENAME=rabbit@localhost"
+
+  # Stop RabbitMQ :
+  # sudo systemctl stop rabbitmq-server
+  - name: Stop RabbitMQ server
+    systemd:
+      name: rabbitmq-server
+      state: stopped
+
+  # Stop epmd:
+  # sudo epmd -kill
+  - name: Stop epmd
+    shell:
+      cmd: /usr/bin/epmd -kill
+
+  # Create a new file /etc/rabbitmq/rabbitmq.conf and add the following lines:
+  # listeners.tcp.local = 127.0.0.1:5672
+  - name: Create /etc/rabbitmq/rabbitmq.conf file
+    shell:
+      cmd: echo "listeners.tcp.local = 127.0.0.1:5672" > /etc/rabbitmq/rabbitmq.conf ; chmod 0644 /etc/rabbitmq/rabbitmq.conf 
+
+   
+  # Create a new file /etc/rabbitmq/advanced.conf and add the following lines:
+  # [
+  #     {kernel,[
+  #         {inet_dist_use_interface,{127,0,0,1}}
+  #     ]}
+  # ].
+  - name: Create /etc/rabbitmq/advanced.conf
+    blockinfile:
+      path: /etc/rabbitmq/advanced.conf
+      create: yes
+      block: |
+        [
+         {kernel,[
+             {inet_dist_use_interface,{127,0,0,1}}
+               ]}
+        ].
+
+  # Now start RabbitMQ again and check if it only listens on localhost for ports 5672 and 25672:
+  # systemctl start rabbitmq-server
+  - name: Start RabbitMQ server
+    systemd:
+      name: rabbitmq-server
+      state: started
+
+  # Add the ‘kat’ vhost
+  # Generate a safe password for the KAT user in rabbitmq. You can use the /dev/urandom method again and put it in a shell variable to use it later:
+  # rabbitmq_pass=$(tr -dc A-Za-z0-9 < /dev/urandom | head -c 20)
+  - name: Generate random password for KAT user in RabbitMQ
+    set_fact:
+      kat_mq_password: "{{ lookup('password', '/dev/null length=20 chars=ascii_letters') }}"
+
+  - name: Show password for debugging purposes (TODO comment these lines)
+    debug:
+      var: kat_mq_password
+
+  # Now create a KAT user for RabbitMQ, create the virtual host and set the permissions:
+  # sudo rabbitmqctl add_user kat ${rabbitmq_pass}
+  - name: Create KAT user for RabbitMQ
+    shell:
+      cmd: "rabbitmqctl add_user kat {{ kat_mq_password }}"
+
+  # sudo rabbitmqctl add_vhost kat
+  - name: Create virtual host
+    shell:
+      cmd: rabbitmqctl add_vhost kat
+
+  # sudo rabbitmqctl set_permissions -p "kat" "kat" ".*" ".*" ".*"
+  - name: Set permissions in RabbitMQ for kat user
+    shell:
+      cmd: rabbitmqctl set_permissions -p "kat" "kat" ".*" ".*" ".*"
+
+  # Now configure KAT to use the vhost we created and with the kat user. To do this, update the following settings for /etc/kat/mula.conf:
+  # SCHEDULER_RABBITMQ_DSN=amqp://kat:<password>@127.0.0.1:5672/kat
+  - name: Configure KAT to use the virtual host
+    lineinfile:
+      path: /etc/kat/mula.conf
+      regexp: "SCHEDULER_RABBITMQ_DSN=.*"
+      line:  "SCHEDULER_RABBITMQ_DSN=amqp://kat:{{ kat_mq_password }}@127.0.0.1:5672/kat"
```

Comment:
This is also `QUEUE_URI` for mula since OpenKAT 1.11, so for mula.conf this must be done the same as for the other configs.